lib/grpcx/server/interceptors/active_record.rb

```diff
@@ -1,5 +1,7 @@
+require 'grpclb/server'
+
 module Grpcx
-  module Server
+  class Server < Grpclb::Server
```

Comment:
I dislike this, but seems no way round it, except, probably, changing naming scheme to smth like `Grpcx::Interceptors::Server::ActiveRecord`